libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1195,7 +1195,7 @@ void RC_Channel::init_aux()
 bool RC_Channel::read_3pos_switch(RC_Channel::AuxSwitchPos &ret) const
 {
     const uint16_t in = get_radio_in();
-    if (in <= 900 or in >= 2200) {
+    if (in <= RC_MIN_LIMIT_PWM or in >= RC_MAX_LIMIT_PWM) {
```

Comment:
my goodness, what is this "or" doing in here?  obviously an existing problem but we should get rid of these.

It looks like the felony was committed by @peterbarker with me as his accomplice! PR https://github.com/ArduPilot/ardupilot/pull/10199

Reply:
changed !